rocAL/rocAL/include/meta_data/meta_data_reader.h

```diff
@@ -57,28 +57,36 @@ struct MetaDataConfig
     unsigned _frame_stride;
     unsigned _out_img_width;
     unsigned _out_img_height;
+    bool _avoid_class_remapping;
+    bool _aspect_ratio_grouping;
 
 public:
-    MetaDataConfig(const MetaDataType& type, const MetaDataReaderType& reader_type, const std::string& path, const std::map<std::string, std::string> &feature_key_map=std::map<std::string, std::string>(), const std::string file_prefix=std::string(), const unsigned& sequence_length = 3, const unsigned& frame_step = 3, const unsigned& frame_stride = 1)
-                    :_type(type), _reader_type(reader_type),  _path(path), _feature_key_map(feature_key_map), _file_prefix(file_prefix), _sequence_length(sequence_length), _frame_step(frame_step), _frame_stride(frame_stride){}
+    MetaDataConfig(const MetaDataType& type, const MetaDataReaderType& reader_type, const std::string& path, const std::map<std::string, std::string> &feature_key_map=std::map<std::string, std::string>(), const std::string file_prefix=std::string(), const unsigned& sequence_length = 3, const unsigned& frame_step = 3, const unsigned& frame_stride = 1, bool avoid_class_remapping = false)
+                    :_type(type), _reader_type(reader_type),  _path(path), _feature_key_map(feature_key_map), _file_prefix(file_prefix), _sequence_length(sequence_length), _frame_step(frame_step), _frame_stride(frame_stride), _avoid_class_remapping(avoid_class_remapping){}
     MetaDataConfig() = delete;
     MetaDataType type() const { return _type; }
     MetaDataReaderType reader_type() const { return _reader_type; }
     std::string path() const { return  _path; }
     std::map<std::string, std::string> feature_key_map() const {return _feature_key_map; }
     std::string file_prefix() const { return  _file_prefix; }
+    bool class_remapping() const { return _avoid_class_remapping; }
+    bool aspect_ratio_grouping() const { return _aspect_ratio_grouping; }
     unsigned sequence_length() const { return _sequence_length; }
     unsigned frame_step() const { return _frame_step; }
     unsigned frame_stride() const { return _frame_stride; }
     unsigned out_img_width() const { return _out_img_width; }
     unsigned out_img_height() const { return _out_img_height; }
     void set_out_img_width(unsigned out_img_width) { _out_img_width = out_img_width; }
     void set_out_img_height(unsigned out_img_height) { _out_img_height = out_img_height; }
+    void set_class_remapping(bool avoid_class_remapping) { _avoid_class_remapping = avoid_class_remapping; }
```

Comment:
The name of API suggests we are setting the class remapping
The name of the variable suggests we are avoiding class remapping

set class remapping would mean we are enabling the remapping right? Not avoiding it. Check how this can be made meaningful

Reply:
Changed it to set_avoid_class_remapping